bin/create

```diff
@@ -68,4 +68,7 @@ var options = {
 
 require('./templates/scripts/cordova/loggingHelper').adjustLoggerLevel(argv);
 
-Api.createPlatform(projectPath, config, options);
+Api.createPlatform(projectPath, config, options).catch(err => {
+    console.error(err);
+    process.exit(2);
+});
```

Comment:
Does this affect our error handler in any fashion?

Reply:
I don't think so, because we check for the patterns in an accumulate of stdout and stderr 🤔

Reply:
We only check for patterns when the `cordova` process finishes with a non-zero code or signal.

In the `cordova plugin` and `cordova build` tasks, we do look for "failure indicative" messages in the output to decide whether it's a success or a failure. But not in `platform add`, which I'm assuming is what this affects.